### ISSUE TITLE
Some pkg recipes refer to outdated archive versions

### ISSUE THREAD

**Issue by chelmuth** [genodelabs/genode-world#181]
Our nightly *build-packages* step builds and publishes all recipes of genode and genode-world but produces the following errors.

```
[2019-06-07 00:48:51] build-packages: publish 'pkg' archives...
[2019-06-07 00:49:07] make[1]: *** [checked_completeness] Error 1
[2019-06-07 00:49:07] make: *** [dependencies_error] Error 2
[2019-06-07 00:49:08] make[1]: *** [checked_completeness] Error 1
[2019-06-07 00:49:08] make: *** [dependencies_error] Error 2
[2019-06-07 00:49:16] make[1]: *** [checked_completeness] Error 1
[2019-06-07 00:49:16] make: *** [dependencies_error] Error 2
[2019-06-07 00:49:19] make[1]: *** [checked_completeness] Error 1
[2019-06-07 00:49:19] make: *** [dependencies_error] Error 2
```

This is due to some pkgs in genode-world referencing explicit versions of archives like follows.

```
world/recipes/pkg/templeos/archives:ehmry/raw/templeos/2017-12-12 [this is available online]
world/recipes/pkg/templeos/archives:genodelabs/src/vbox5-nova/2018-08-28
world/recipes/pkg/templeos/archives:genodelabs/src/base-nova/2018-08-22
world/recipes/pkg/templeos/archives:genodelabs/src/libc/2018-07-26
world/recipes/pkg/templeos/archives:genodelabs/src/init/2018-08-21
world/recipes/pkg/templeos/archives:genodelabs/src/nit_fb/2018-07-12
world/recipes/pkg/templeos/archives:genodelabs/src/libc/2018-07-26
world/recipes/pkg/templeos/archives:genodelabs/src/posix/2018-07-25
world/recipes/pkg/templeos/archives:genodelabs/src/zlib/2018-07-09
world/recipes/pkg/templeos/archives:genodelabs/src/nit_fb/2018-07-12
world/recipes/pkg/templeos/archives:genodelabs/src/libiconv/2018-07-09
world/recipes/pkg/templeos/archives:genodelabs/src/stdcxx/2018-07-09
world/recipes/pkg/templeos/archives:genodelabs/src/vfs/2018-08-28
world/recipes/pkg/flif_capture/archives:genodelabs/src/stdcxx/2019-02-25
world/recipes/pkg/flif_capture/archives:genodelabs/src/init/2019-03-17
world/recipes/pkg/flif_capture/archives:genodelabs/src/nit_fb/2019-03-17
world/recipes/pkg/flif_capture/archives:genodelabs/src/nitpicker/2019-03-17
world/recipes/pkg/flif_capture/archives:genodelabs/pkg/themed_wm/2019-03-18
world/recipes/pkg/terminal_editor/archives:genodelabs/raw/qt5_dejavusans/2018-09-06
world/recipes/pkg/terminal_editor/archives:genodelabs/src/expat/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/freetype/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/jpeg/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/libc/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/libpng/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/mesa/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/pcre16/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_component/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_core/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_gui/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_printsupport/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_qjpeg/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_qpa_nitpicker/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/qt5_widgets/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/stdcxx/2018-11-26
world/recipes/pkg/terminal_editor/archives:genodelabs/src/vfs/2018-11-27
world/recipes/pkg/terminal_editor/archives:genodelabs/src/zlib/2018-11-26
```

Could we remove the explicit version spec to silence the errors or do we have to adapt our nightly to support this case (e.g., by downloading missing dependencies)?

@ehmry you contributed these packages originally and may have an opinion

**Comment by ehmry** [comment]
Fixed in PR #182.

**Comment by chelmuth** [comment]
Thanks.
